### PR TITLE
fix checkbox sizing

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/server/issues/41262

| Before | After |
|---|---|
| image | image |

### CODE REVIEW

**Review by JuliaKirschenheuter — APPROVED** [review]
😊

**Review by raimund-schluessler — CHANGES_REQUESTED** [review]
This breaks the height of `NcCheckBoxRadioSwitch` which should be 44 px:

Before:

Screenshot 2023-11-06 at 16-59-58 Nextcloud Vue Style Guide
After:
Screenshot 2023-11-06 at 17-00-19 Nextcloud Vue Style Guide

**Review by susnux — APPROVED** [review]

File: src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue [hunk 1]
```diff
@@ -208,7 +208,8 @@ export default {
 	user-select: none;
 	min-height: $clickable-area;
 	border-radius: $clickable-area;
-	padding: 4px $icon-margin;
+	box-sizing: content-box !important;
```
Comment (susnux) [inline src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue:211]: Not sure if this is the best solution, we could also set `min-width: $clickable-area;` and set the padding to `padding: 4px 10px`. That would not require an `!important`.

**Comment by szaimen** [comment]
Ready for re-review 👍

**Review by susnux — APPROVED** [review]

**Comment by susnux** [comment]
But please squash the commits

**Comment by szaimen** [comment]
> But please squash the commits

Can you do that please? I currently cannot check out the vue library at all due to
image

**Review by JuliaKirschenheuter — COMMENTED** [review]
Is it known that checkbox i moved to left now? Or another issue?

Screenshot from 2023-11-16 15-30-08

**Comment by JuliaKirschenheuter** [comment]
master:

Screenshot from 2023-11-16 15-38-28

**Comment by raimund-schluessler** [comment]
> > But please squash the commits
> 
> Can you do that please? I currently cannot check out the vue library at all due to image

Although unrelated to this PR: These files should not be there. I removed them in https://github.com/nextcloud-libraries/nextcloud-vue/pull/4837.

**Comment by szaimen** [comment]
> But please squash the commits

done 👍

**Comment by Antreesy** [comment]
Label text is now falling out of the flex container borders:
image
With flex-shrink reverted for `checkbox-content__text` class:
image

**Pull request merged**